src/Http/Session/Session.php

```diff
@@ -0,0 +1,260 @@
+<?php
+
+namespace Framework3\Http\Session;
+
+use Framework3\Bag\Bag;
+use Framework3\Security\UserInterface;
+
+class Session implements SessionInterface
```

Comment:
Codacy found an issue: [The class Session has 11 non-getter- and setter-methods. Consider refactoring Session to keep number of methods under 10.](https://app.codacy.com/gh/pierregaimard/framework3/pullRequest?prid=6299285)